.travis.yml

```diff
@@ -1,4 +1,18 @@
 language: node_js
 node_js:
-    - "0.10"
-    - "0.12"
+    - 0.1
+    - 0.12
+notifications:
+    email:
+        recipients:
+            - ypt-team@yahoo-inc.com
+        on_success: change
+        on_failure: always
+deploy:
+    provider: npm
+    email: ypt-team@yahoo-inc.com
+    api_key: 
+        secure: s5KbfZ2QmDElovpw8PZYFwiwYIcvmORO1dpnsbSpuNrdgW/vr79vxmETg1RdaTOC+eprtJgf3DkqBeuUeavc2S9ksO9sbryhL48Sfu3wQBQWIfana6H1weVpHxUfS4TU9bT2zRlxRDQsYuFmmtacaJ0DpDKrnpdpVBgXxKJWq4o=
+    on:
+        tags: true
```

Comment:
This will only trigger an npm publish when we tag a release on github.

This may be a better workflow then listening for package.version changes since it prevents us for getting in the habit of not tagging.

/cc @ericf @caridy - thoughts?